We do not need to check typing for this project.

--- wrapt-2.2.1/tests/conftest.py.orig
+++ wrapt-2.2.1/tests/conftest.py
@@ -136,6 +136,8 @@
     """
 
     def collect(self):
+        return
+
         # Only run this custom collection on Python 3.10+
         if version < (3, 10):
             return
